operacje_na_listach/Function_for_exponentiation.py

```diff
@@ -0,0 +1,13 @@
+def power(list):
```

Comment:
please change name of `list`, `list` is a built-in function in python, you can use it like that to create an empty list:
```py
some_list = list()
```
If you name your variable like that then you won't be able to use `list` method in this function (it will be "shadowed")

Reply:
you can rename it to `list_`